src/commands/UploadReactNativeCommand.ts

```diff
@@ -163,7 +163,7 @@ const reactNativeFetchOpts = [
   {
     name: 'bundler-entry-point',
     type: String,
-    description: 'the entry point of your React Native app',
+    description: 'the entry point file of your React Native app',
```

Comment:
Might be worth saying that the default value is `index.js` in this description?

Reply:
I think that's a good idea and should make it a lot clearer, but should we then add defaults to all of the options? It seems a bit weird to only mention the default for one option

How should the default be formatted? e.g.

```suggestion
    description: 'the entry point file of your React Native app (defaults to index.js)',
```

Reply:
> should we then add defaults to all of the options

I guess so. Thankfully there aren't too many that have default values, I think it's just the `--fetch`-specific ones, `--endpoint` and `--project-root`.

The format you suggested looks good.

Reply:
👍 Yeah, that's it

I've left `--endpoint` without a default as it's different for Browser/Node vs React Native and the default doesn't seem that relevant